src/Pages/Essence/Essence.module.css

```diff
@@ -0,0 +1,3 @@
+html {
```

Comment:
Oh nu ser jag du kan ta bort rad 1-3. Alexandra har redan lagt in den under pages så den kommer att dyka upp för dig också sen när du har lagt in i develop. Så i din .container på rad 7 så borde du kunna använda background-color: var(--esence-bg-color); så kommer den synas efteråt i develop.

Reply:
Aa det stämmer:)